pgx-pg-config/src/lib.rs

```diff
@@ -119,49 +119,45 @@ impl PgConfig {
         self.path().unwrap().parent().unwrap().to_path_buf()
     }
 
+    fn parse_version(&self) -> eyre::Result<(u16, u16)> {
+        let version_string = self.run("--version")?;
+        let version_parts = version_string.split_whitespace().collect::<Vec<&str>>();
+        let version = version_parts
+            .get(1)
+            .ok_or_else(|| eyre!("invalid version string: {}", version_string))?
+            .split('.')
+            .collect::<Vec<&str>>();
+        if version.len() < 2 {
+            return Err(eyre!("invalid version string: {}", version_string));
+        }
+        let major = u16::from_str(version[0])
+            .map_err(|e| eyre!("invalid major version number `{}`: {:?}", version[0], e))?;
+        let mut minor = version[1];
+        let digits = HashSet::from(['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']);
+        let mut end_index = minor.len();
+        for (i, c) in minor.chars().enumerate() {
+            if !digits.contains(&c) {
```

Comment:
`if !c.is_ascii_digit()` (https://doc.rust-lang.org/nightly/std/primitive.char.html#method.is_ascii_digit) and removing `digits` is cleaner IMO

Reply:
Didn't know about that stdlib function. That is cleaner. Thanks!